src/_experimental.scss

```diff
@@ -4,6 +4,16 @@
 	- Experimental  CSSspecific for view port are defined in the "views/[view port]" theme CSS
 */
 
+// $alert-green--color: #2b542c;
+$danger-color: #d3080c;
```

Comment:
Add a comment before to mention those new color is for the provisional alert.
For each those new SASS variable, can you prefix them with "provisional" ? It is just to avoid possible conflict with existing code.